CondTools/RunInfo/test/test_lhcInfoNewPopCon.sh

```diff
@@ -50,11 +50,11 @@ cmsRun ${SCRIPTS_DIR}/LHCInfoPerLSPopConAnalyzer.py mode=endFill \
     destinationConnection="sqlite_file:lhcinfo_pop_unit_test.db" \
     startTime="2022-07-11 22:00:00.000" endTime="2022-07-12 18:10:10.000" \
     tag=ls_end_test2 > ls_end_test2.log || die "cmsRun LHCInfoPerLSPopConAnalyzer.py mode=endFill" $?
-assert_equal 69 `cat ls_end_test2.log | grep -E '^Since ' | \
+assert_equal 70 `cat ls_end_test2.log | grep -E '^Since ' | \
```

Comment:
@JanChyczynski the error in the unit test is already there in the IB, therefore it was not originated by this PR.
Could you please summarize here why was this fix needed? Is this number of payloads fixed to be "70" in all cases now?

Reply:
Ping @JanChyczynski 
If we solve this point, this PR can be merged, and the backport will follow

Reply:
Ok, let consider https://github.com/cms-sw/cmssw/issues/42828#issuecomment-1727300713 as an answer, and move forward with this PR